packages/dx-react-common/docs/guides/performance-optimization.md

```diff
@@ -0,0 +1,29 @@
+# React Components - Performance Optimization
+
+This article describes the main rules you should follow to provide high perfromance for your application. For more information on performance optimization, refer to the [React guidelines](https://reactjs.org/docs/optimizing-performance.html).
```

Comment:
perfromance - SP

Reply:
"the main rules you should follow to provide" AWK - How can you provide high performance?

Reply:
For more information on performance optimization, refer to the [React guidelines](https://reactjs.org/docs/optimizing-performance.html) - WO